### PR TITLE
Modification of FPS parameter

### PR DESCRIPTION
**Pull request by mss-park**
FPS parameter is changed according to the  #109

### CODE REVIEW

**Review by dkozinski — COMMENTED** [review]
Please consider using structure for fps representation instead of using two variables representing fps numerator and fps denominator. 
In FFmpeg for framerate definition AVRational type is used.

typedef struct XeveRational {
    int num; ///< Numerator
    int den; ///< Denominator
} XeveRational;

struct _XEVE_PARAM {
...
    XeveRational fps;
...
}

**Comment by mss-park** [comment]
The structure is added. Please check the updated code.

**Review by dkozinski — APPROVED** [review]

**Comment by dkozinski** [comment]
> The structure is added. Please check the updated

**Comment by dkozinski** [comment]
Looks good. Thanks

File: src_base/xeve_enc.c [hunk 1]
```diff
@@ -2618,7 +2618,7 @@ void xeve_param2string(XEVE_PARAM * param, char * sei_buf, int padx, int pady)
     sei_buf += snprintf(sei_buf, max_n, "profile=%d", param->profile);
     sei_buf += snprintf(sei_buf, max_n, " threads=%d", param->threads);
     sei_buf += snprintf(sei_buf, max_n, " input-res=%dx%d", param->w - padx, param->h - pady);
-    sei_buf += snprintf(sei_buf, max_n, " fps=%u", param->fps);
+    sei_buf += snprintf(sei_buf, max_n, " fps=%u", param->fps.num/ param->fps.den);
```
Comment (mpeg5) [inline src_base/xeve_enc.c:2621]: this may need type casting param->fps.num to (float)param->fps.num

**Review by mpeg5 — COMMENTED** [review]
please find my inline comment

File: src_base/xeve_enc.c (hunk 1 above)
Comment (mss-park) [reply src_base/xeve_enc.c]: Type casting is added.

**Review by mpeg5 — COMMENTED** [review]
LGTM

**Pull request merged**